### PR TITLE
Feat/add mlhub labels

### PR DESCRIPTION
**Pull request by chorng**
* add microsoft floods collection
* add south africa crop type competition collection
* add plantvillage crop type kenya collection
* add western usa live fuel moisture collection

### CODE REVIEW

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

**Comment by chorng** [comment]
@Dorothyrono Fixed the typos and doubled check. Please review again.

File: collections/south-africa-crops-competition.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+Name: South Africa Crop Type Competition
+Description: |
+  The South Africa Crop Type Competition is a dataset produced as part of the [Radiant Earth Spot the Crop Challenge](https://zindi.africa/hackathons/radiant-earth-spot-the-crop-hackathon). This dataset collection contains the field identification lable which represents the area of crop field and the corresponding crop type label collected via aerial and vehicle surveys.
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/south-africa-crops-competition.yaml]: doubel check, not resolved

File: collections/south-africa-crops-competition/README.MD [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# South Africa Crop Type Competition
+
+## Short description
+
+The South Africa Crop Type Competition is a dataset produced as part of the [Radiant Earth Spot the Crop Challenge](https://zindi.africa/hackathons/radiant-earth-spot-the-crop-hackathon). This dataset collection contains the field identification lable which represents the area of crop field and the corresponding crop type label collected via aerial and vehicle surveys.
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/south-africa-crops-competition/README.MD]: double check, not resolved

File: collections/western-usa-fuel-moisture.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+Name: Western USA Live Fuel Moisture
+Description: |
+  The Western USA Live Fuel Moisture is a dataset manually collected in the western United States by the United States Forest Sercice. Live fuel moisture represents the mass of water in live vegetation elements likes leaves, needles, and twigs divided by its oven-dried mass in percentage. It is an indicator which can be used for wildfire assessment. This collection is rasterized from the orginal vector data (points) at 10 m resolution to be paired with Sentinel-1 and Sentinel-2 imagery.
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/western-usa-fuel-moisture.yaml]: doubel check, not fully resolved

File: collections/western-usa-fuel-moisture.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      Columns:
+        - Name: Name
+          Title: Name
+        - Name: Description
+          Title: Description
+      Rows:
+        - Name: water
+          Description: flood extent
+Contact: kkrao.j@gmail.com
+Provider: "Stanford University"
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/western-usa-fuel-moisture.yaml]: Looks better with `and`

"Stanford University and [Source Cooperative](https://beta.source.coop/repositories/stanford/sar-moisture-conent/description)"

File: collections/western-usa-fuel-moisture/README.MD [hunk 5]
```diff
@@ -0,0 +1,71 @@
+# Western USA Live Fuel Moisture
+
+## Short description
+
+The Western USA Live Fuel Moisture is a dataset manually collected in the western United States by the United States Forest Sercice. Live fuel moisture represents the mass of water in live vegetation elements likes leaves, needles, and twigs divided by its oven-dried mass in percentage. It is an indicator which can be used for wildfire assessment. This collection is rasterized from the orginal vector data (points) at 10 m resolution to be paired with Sentinel-1 and Sentinel-2 imagery.
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/western-usa-fuel-moisture/README.MD]: dobel check, not fully resolved

**Comment by chorng** [comment]
Done.

**Review by Dorothyrono — APPROVED** [review]

**Comment by Dorothyrono** [comment]
Looks good. Can merge

**Pull request merged**